src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -79,12 +79,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         with self.argument_context(scope, arg_group='Network Rule') as c:
             c.argument('default_action', arg_type=get_enum_type(DefaultAction),
                        help='Default action to apply when no rule matches. Only applicable to Premium SKU.')
+            c.argument('public_network_access_enabled', get_three_state_flag(), help="Allow public network access for the container registry. The Default is allowed")
```

Comment:
I saw a few other services used `--enable-public-network`, should we be consistent?

https://github.com/Azure/azure-cli/blob/3315616902b390805ca8d138a6fa168ff0518caf/src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py#L67

https://github.com/Azure/azure-cli/blob/015be59dfb57fbaf9d5215d49fe0488ea41d3a53/src/azure-cli/azure/cli/command_modules/sql/_params.py#L1134

Reply:
Agreed, let us be consistent

Reply:
Per conversation with Mario from NRP, we can use `--public-network-enabled` for consistency with other ACR arguments